skyreader/plot/plot.py

```diff
@@ -771,9 +771,9 @@ def bounding_box(ra, dec, theta, phi):
             ras = list(np.asarray(saving_contours[i][0]).T[0])
             decs = list(np.asarray(saving_contours[i][0]).T[1])
             tab = {"ra (rad)": ras, "dec (rad)": decs}
-            savename = unique_id + ".contour_" + val + ".txt"
+            savename = self.output_dir / f"{unique_id}.contour_{val}.txt"
             try:
-                LOGGER.info("Dumping to {savename}")
+                LOGGER.info(f"Dumping to {savename}")
                 ascii.write(tab, savename, overwrite=True)
```

Comment:
This is no longer a name, so `savepath` would be more fitting. I presume `ascii.write()` works with a `Path` object.